x11/makedepend/Portfile

```diff
@@ -22,9 +22,10 @@ homepage            https://www.x.org/
 master_sites        http://xorg.freedesktop.org/archive/individual/util/
 use_bzip2           yes
 
-checksums           sha1    2599afa039d2070bae9df6ce43da288b3a4adf97 \
-                    rmd160  d1ba5b107ae145608a5f3e932fa2794eec1ddd0a \
-                    sha256  f7a80575f3724ac3d9b19eaeab802892ece7e4b0061dd6425b4b789353e25425
+checksums           sha1    3f321f7b570ec98f1fabc441267d29347a2e1456 \
+                    rmd160  6e6aa137d6cc63ae03feea0d45b4c51d4105ee60 \
+                    sha256  d558a52e8017d984ee59596a9582c8d699a1962391b632bec3bb6804bf4d501c \
+                    size	147616
```

Comment:
Please try to avoid tabs.

(Note that if you delete the checksums and run `sudo port -v checksum` (or `extract`), you also get the desired checksums printed.)

Reply:
Done. Thank you for the tip.